server/raft_transport.go

```diff
@@ -190,8 +190,10 @@ func (t *rpcTransport) processQueue(raftNodeID proto.RaftNodeID) {
 		}
 
 		if !client.IsHealthy() {
-			log.Warningf("raft client for node %d unhealthy", nodeID)
-			return
+			if log.V(1) {
```

Comment:
(should add that the old code would drop the outgoing message queue, but if you miss a couple of Raft messages, then that's likely not a bad idea anyway).